master/getting-started/kubernetes/upgrade.md

```diff
@@ -40,40 +40,23 @@ This section covers upgrading a [self-hosted]({{site.baseurl}}/{{page.version}}/
 Note that while a self-hosted installation of Calico is typically done all at once (via calico.yaml), it is
 recommended to perform upgrades one component at a time.
 
-#### Upgrading the Calico policy controller
+#### Upgrading the Calico kubernetes controllers (previously kube-policy-controller)
```

Comment:
I would omit the parenthetical...it is confusing, especially in a heading. If you wanted to mention this, you could do so in the body. 

The confusion is exacerbated by a lack of parallel construction: "kube-policy-controller" does not match "kubernetes controllers"...both in terms of quantity (one is plural, the other is not) and type of noun (one is the name of an actual container, the other is not)

All this is just to elaborate on why I think you should delete the parenthetical, but also, "kubernetes" --> "Kubernetes"